eurovision/notebooks/story.ipynb

```diff
@@ -818,7 +817,8 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "These are the countries with the best average score (though we only count countries who have participated at least 5 times in the finals)."
+    "Winning is one thing, but which countries perform best _on average_? To make this statistic a bit more robust, we only look at countries which have participate at least five times (see `count` variable).\n",
```

Comment:
The reference to the `count` variable is nice too, drawing the reader's attention to the code rather than just leaving it for them to figure out (or skip past).

The question comes up "does it actually make any difference"? (but because it's code, I could immediately check that myself, which is neat).